homeassistant/components/octoprint/config_flow.py

```diff
@@ -80,6 +84,7 @@ async def async_step_user(self, user_input=None):
                         user_input[CONF_PORT],
                         user_input[CONF_PATH],
                         user_input[CONF_SSL],
+                        user_input.get(CONF_VERIFY_SSL, True),
```

Comment:
```suggestion
                        user_input[CONF_VERIFY_SSL],
```

If I see this correctly this new option is required and has a default, thus it always exists.